[2024-02-25T13:09:54.108+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-29T00:00:00+00:00 [queued]>
[2024-02-25T13:09:54.126+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-29T00:00:00+00:00 [queued]>
[2024-02-25T13:09:54.126+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2024-02-25T13:09:54.219+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): second_task> on 2021-12-29 00:00:00+00:00
[2024-02-25T13:09:54.238+0000] {standard_task_runner.py:57} INFO - Started process 1731 to run task
[2024-02-25T13:09:54.252+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-29T00:00:00+00:00', '--job-id', '458', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpwwyboth6']
[2024-02-25T13:09:54.255+0000] {standard_task_runner.py:85} INFO - Job 458: Subtask second_task
[2024-02-25T13:09:54.455+0000] {task_command.py:410} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-29T00:00:00+00:00 [running]> on host 0c40846c7775
[2024-02-25T13:09:54.841+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-12-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-29T00:00:00+00:00'
[2024-02-25T13:09:54.843+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-02-25T13:09:54.843+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-02-25T13:09:54.870+0000] {subprocess.py:86} INFO - Output:
[2024-02-25T13:09:54.874+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-02-25T13:09:54.875+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-02-25T13:09:54.943+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211229T000000, start_date=20240225T130954, end_date=20240225T130954
[2024-02-25T13:09:55.042+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-25T13:09:55.092+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
